etc/containerpilot.json

```diff
@@ -49,9 +49,7 @@
     {{ if and .CONSUL_AGENT .ACME_DOMAIN }},{{ end }}
     {{ if .ACME_DOMAIN }}
     {
-      "command": ["/usr/local/bin/consul-template",
-                  "-config", "/etc/acme/watch.hcl",
-                  "-consul", "{{ if .CONSUL_AGENT }}localhost{{ else }}{{ if .CONSUL }}{{ .CONSUL }}{{ else }}consul{{ end }}{{ end }}:8500"],
+      "command": ["acme", "watch"],
```

Comment:
We certainly can. I just did it this way so that all the moving parts related to certificate acquisition was contained within the `acme` script, so that when you're looking at that you see it all, versus some pieces here, some there. Things like the `CONSUL_AGENT` are tucked away cleanly in the `acme` script, etc. Cleans up `containerpilot.json` a bit. Admittedly, this is purely stylistic.